draft-ietf-rats-eat.md

```diff
@@ -1494,8 +1494,8 @@ A profile can apply to Attestation Evidence or to Attestation Results or both.
 
 A profile document doesn't have to be in any particular format. It may be simple text, something more formal or a combination.
 
-In some cases CDDL may be created that replaces CDDL in this or other document to express some profile requirements.
-For example, to require the altitude data item in the location claim, CDDL can be written that replicates the location claim with the altitude no longer optional.
+In some cases it may be desirable for CDDL to be created that replaces CDDL in this or other document to express some profile requirements.
```

Comment:
```suggestion
In some cases it may be desirable for CDDL to be created that constrains CDDL in this or other document to express some profile requirements.
```

Reply:
I like the focus on actual "profiling", which literally means "constraining" to a subset. Thank you Thomas, for that clarification. I am seconding this suggestion.